en/services/manual_control.md

```diff
@@ -0,0 +1,88 @@
+# Manual Control Protocol
+
+The Manual Control Protocol enables controlling a system using standard joystick axes nomenclature, along with a joystick-like input device. 
+ 
+The protocol is implemented with just the [`MANUAL_CONTROL`](../messages/common.md#MANUAL_CONTROL) message.
+It consists of the `target` system to be controlled, four primary axes (`x`, `y`, `z`, `r`), two extension axes (`s`, `t`), and two 16-bit fields to represent the states of up to 32 buttons (`buttons`, `buttons2`).
+Unused axes can be disabled, and the extension axes must be explicitly enabled using bits 0 and 1 of the `enabled_extensions` field.
+
+## Mapping Axes
+
+Manual control is performed in the vehicle-frame. All axis values are normalised to the range -1000 to 1000.
+
+### Rotation-Focused Control
+
+The typical axis assignments for a thrust- and rotation-controlled vehicle (e.g. planes, multicopters) are listed below.
+
+field | motion axis | +ve direction | -ve direction
+-- | -- | -- | --
+`x` | pitch | forward/nose-down | backward/nose-up
+`y` | roll | right-down | left-down
+`z` | thrust | positive | negative
+`r` | yaw | counter-clockwise | clockwise
+
+### Directional Control
+
+Vehicles with direct control over vehicle translation directions (multicopters) typically use the following mappings.
+
+field | motion axis | +ve direction | -ve direction
+-- | -- | -- | --
+`x` | forward | forward | backward
+`y` | lateral | right | left
+`z` | vertical | up | down
+`r` | yaw | counter-clockwise | clockwise
+`s` | pitch | forward/nose-down | backward/nose-up
+`t` | roll | right-down | left-down
+
+## Mapping Buttons
+
+Button functions are vehicle-specific and determined in the vehicle firmware.
+They are generally user-configurable using firmware parameters (e.g. ArduCopter's [`BTN_FUNCn`](https://ardupilot.org/copter/docs/parameters.html#btn-func1-button-pin-1-rc-channel-function) or ArduSub's [`BTNn_FUNCTION`](https://www.ardusub.com/developers/full-parameter-list.html#btnnfunction-function-for-button)), through the [Parameter](./parameter.md) or [Extended Parameter](./parameter_ext.md) protocols.
```

Comment:
On quick scan of code it looks like PX4 maps a subset of _fixed functions_ to the message button slots. Less flexible if you have more functions you want to expose than slots. Worth noting this here though right?

Reply:
I just had a quick poke around and got lost after the buttons get passed to `_rc_pub` - not sure if the functions are indeed fixed-mapped to slots or if they're adjustable like with Ardu*. If they are definitely fixed then it makes sense to note here as a difference - agreed.